src/utils/helpers/list/useGetFilterDefinitions.ts

```diff
@@ -59,46 +61,45 @@ export const useApplicationFilters = () => {
     },
   }
 
-  const APPLICATION_FILTERS: FilterDefinitions = {
+  const FILTER_DEFINITIONS: FilterDefinitions = {
     lastActiveDate: {
       type: 'date',
+      default: defaultFilters.some((filter) => filter === 'lastActiveDate'),
+      visibleTo: checkFilterVisibility("LAST_ACTIVE_DATE"),
       title: strings.FILTER_LAST_ACTIVE,
       options: { namedDates: NAMED_DATE_RANGES },
     },
-    // isFullyAssignedLevel1: {
-    //   type: 'boolean',
-    //   title: strings.FILTER_ASSIGNMENT_STATUS,
-    //   options: {
-    //     booleanMapping: {
-    //       true: strings.FILTER_ASSIGNMENT_STATUS_TRUE,
-    //       false: strings.FILTER_ASSIGNMENT_STATUS_FALSE,
-    //     },
-    //   },
-    // },
-    // reviewersAssignedCount: {
-    //   type: 'number',
-    // },
     applicantDeadline: {
       type: 'date',
+      default: defaultFilters.some((filter) => filter === 'applicantDeadline'),
+      visibleTo: checkFilterVisibility("DEADLINE_DATE"),
       title: strings.FILTER_APPLICANT_DEADLINE,
       options: { namedDates: NAMED_DATE_RANGES },
     },
     type: {
       type: 'equals',
+      default: defaultFilters.some((filter) => filter === 'type'),
+      visibleTo: checkFilterVisibility("TYPE"),
       options: { substituteColumnName: 'templateCode' },
     },
     status: {
       type: 'enumList',
+      default: defaultFilters.some((filter) => filter === 'status'),
+      visibleTo: checkFilterVisibility("STATUS"),
       title: strings.FILTER_STATUS,
       options: { enumList: Object.values(ApplicationStatus) },
     },
     outcome: {
       type: 'enumList',
+      default: defaultFilters.some((filter) => filter === 'outcome'),
+      visibleTo: checkFilterVisibility("OUTCOME"),
       title: strings.FILTER_OUTCOME,
       options: { enumList: Object.values(ApplicationOutcome) },
     },
-    orgName: {
+    org: {
```

Comment:
Oh -also this breaks the list -- the query to the back-end no longer works, so we'll need to change this back to "orgName"